packages/article/src/link-info.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+
+export interface LinkInfo {
+  url: string;
+  type: string;
```

Comment:
if this can be only `'article'` or `'link'`, it'd be better to define it as `type: 'article' | 'link'`

Reply:
Fixed 9e95801